src/OrderTaking/PlaceOrder.Implementation.fs

```diff
@@ -121,315 +121,315 @@ type CreateEvents =
 // ======================================================
 // Section 2 : Implementation
 // ======================================================
+module internal Implementation =
```

Comment:
Submodule created that keeps implementation internal. All changes below this are just do to indentation (no real code changes).